README.md

```diff
@@ -34,19 +35,27 @@ Lastly, a Dockerfile is provided for installing pre-requisite packages & launchi
 
 Please ensure you meet the [pre-requisites](https://rapids.ai/start.html#prerequisites)
 
-Our container extends the base notebooks container. To build:
+Our container extends the base notebooks container. Unlike our Notebooks repo, which comes with the container, Notebooks Extended is meant to be a constatly updating source of community contributions.  We've made it easy to include Notebooks Extended in your RAPIDS container with 3 easy steps:
```

Comment:
constatly -> constantly

Optional/suggestion: I'd change "We've made it easy to include Notebooks Extended in your RAPIDS container with 3 easy steps:" to something like
 "You can run Notebooks Extended in a container with 3 steps:" (sounds a little less marketing)